app/src/main/res/layout/content_main_p.xml

```diff
@@ -20,19 +20,29 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        >
+        android:id="@+id/constraint_main"
+        android:background="@drawable/back_cover">
```

Comment:
You have not formatted all your files using Square's code style as mentioned in the repo's docs. You just need to press `Ctrl + Shift + L` to autoformat all the lines.